src/main/java/sirius/web/health/CachingLoadInfoProvider.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.health;
+
+import sirius.kernel.cache.CacheManager;
+import sirius.kernel.cache.InlineCache;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ * Provides a basic implementation which caches all computed infos for up to ten seconds.
+ * <p>
+ * Be aware, that subclasses have to specify <tt>LoadInfoProvider</tt> in their <tt>Register</tt> annotation:
+ * {@code @Register(classes = LoadInfoProvider.class)}.
+ */
+public abstract class CachingLoadInfoProvider implements LoadInfoProvider {
```

Comment:
"CachedLoadInfoProvider"?
Or els it sounds like it would provide some kind of info about the "caching load", but it is supposed to be the cached load infos, if i am not mistaken

Reply:
Well it is a load info provider which does caching :) Otherwise is would be a CacheLoadInfoProvider (?)

Reply:
ok :)